doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,923 @@
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "resp = requests.post(\"http://localhost:8000/Adder/\", json={\"array\": [40]})\n",
+    "resp.raise_for_status()\n",
+    "resp.json()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Nice! We sent `[40]` as our input and got `[42]` as our output in JSON format.\n",
+    "\n",
+    "You can also specify multi-dimensional array in the JSON payload, as well as \"dtype\" and \"shape\" field to process to array. For more information about the array input schema, see [Ndarray](serve-ndarray-schema).\n",
+    " \n",
+    "That's it for array! Let's take a look at tabular input."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Predictor accepting Pandas DataFrame\n",
+    "Let's now take a look at a predictor accepting dataframe input. We'll perform some simple column wise transformation on the input data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "\n",
+    "class DataFramePredictor(Predictor):\n",
+    "    \"\"\"Dummy predictor that first multiplies input then increment it.\"\"\"\n",
+    "    def __init__(self, increment: int):\n",
+    "        self.increment = increment\n",
+    "\n",
+    "    @classmethod\n",
+    "    def from_checkpoint(cls, ckpt: Checkpoint):\n",
+    "        return cls(ckpt.to_dict()[\"increment\"])\n",
+    "\n",
+    "    def predict(self, inp: pd.DataFrame) -> pd.DataFrame:\n",
+    "        inp[\"prediction\"] =  inp[\"base\"] * inp[\"multiplier\"] + self.increment\n",
+    "        return inp\n",
+    "\n",
+    "local_df_predictor = DataFramePredictor.from_checkpoint(local_checkpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just like the `AdderPredictor`, we'll use the same `ModelWrapperDeployment` approach to make it queryable with HTTP. \n",
+    "\n",
+    "You might noticed a small addition this time. We added `http_adapter=pandas_read_json` as the keyword argument. This tells Serve how to parse incoming JSON request into a DataFrame. The `pandas_read_json` adapter accepts:\n",
+    "- [Pandas-parsable JSON](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html) in HTTP body\n",
+    "- Optionally keyword arguments to [`pandas.read_json`](https://pandas.pydata.org/docs/reference/api/pandas.read_json.html) function via HTTP url parameters.\n",
+    "\n",
+    "To learn more, see [HTTP Adapters](serve-http-adapters).\n",
+    "\n",
+    "```{note}\n",
+    "You might wonders why does the previous array predictor doesn't need to specify any http adapter. This is because Serve default to a built-in adapter called `json_to_ndarray`(ray.serve.http_adapters.json_to_ndarray)!\n",
```

Comment:
```suggestion
    "You might wonder why the previous array predictor doesn't need to specify any http adapter. This is because Ray Serve defaults to a built-in adapter called `json_to_ndarray`(ray.serve.http_adapters.json_to_ndarray)!\n",
```